build-scripts/for-linux/prepare-debian-for-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+DIR=`dirname $0`
+$DIR/../for-linux/prepare-debian-based.sh wx32 amd64
+
+# Install requirements for tests infrastructure
+sudo DEBIAN_FRONTEND=noninteractive apt-get install -y gcovr
+
+# Install cpptrace from sources
+if ! ls /usr/local/lib | grep libcpptrace &> /dev/null
```

Comment:
```suggestion
if ! ldconfig -p | grep libcpptrace &> /dev/null
```

because the library might be installed in another dir

Reply:
@oleg68 I've tested ldconfig, but it does not produce the result expected. I will try once more time

Reply:
> @oleg68 I've tested ldconfig, but it does not produce the result expected. I will try once more time

Could you give more information. What case ldconfig does provide incorrect result in?

Reply:
I'm not in front of my computer, but the output does not seem to find the lib.

Reply:
@oleg68 I don't know why but lpptrace is not in the list with ldconfig

Reply:
I don't know if that helps but the file is named `libcpptrace.a`

Reply:
It helps. ldconfig searches only .so files, not .a ones.